cha/req.tex

```diff
@@ -0,0 +1,89 @@
+
+The elicitated requirements will be divided into two categories: functional requirements and UI/UX requirements.
+We start with describing the technical requirements which can be elicitated from \cref{part:introduction}.
+
+\begin{eletterate}
+    \item \textbf{Run on Android 5.0+} \hfill\\
+        As mentioned in the problem statement in \cref{cha:problem_statement}, Android is chosen since it is the available system in our project group.
+        This means that the app have to run on Android five and later versions.
+        Version five is chosen since significant changes happened to the Android sound stack in version five\cite{android_5_sound_stack}.
+
+    \item \textbf{At least two devices\tnnote{device (written in reqiurements) vs smartphone (written in problem statement)} must be able to connect to the same session wirelessly.}\tnnote{Session is first mentioned here.} \hfill\\
+        To be able to achieve multiple playback across multiple devices, as mentioned in \cref{cha:problem_statement},
+        then two or more devices have to be able to connect to each other wirelessly.
+        They connect through a ``session'', which is a master/slave relation, where only one device is the master, the rest are slaves.
+        This session makes it possible to transmit data from the master to the connected slaves.
+        The device which creates the session is automatically the master, slaves can find already created sessions and connect to one.
+
+    \item \textbf{The device which start the session, i.e. the master, must be able to control the playback, e.g. pause, resume and change media, on all devices.} \hfill\\
+        This requirement relates to the use cases described in \cref{sec:category_increase_volume_and_area}\tnnote{This needs to be added explicit in use case.}.
+        The master is in control of the created session, including the playback of the media.
+        This means that the master device can pause and resume the playing media for the connected slaves.
+        Furthermore the master can also change the media being played, skip media or completely stop the playback of media.
+
+    \item \textbf{Each user must be able to change their own volume.} \hfill\\
+        If the user do not want to listen to the music as loud as the rest connected to the session, as mentioned in \cref{sec:category_increase_volume_and_area}, 
+        then a user must be able to control and change the volume his device when it is connected to a session.
+        This possibility applies for both the master device and the connected slaves.
+        The change must not have an effect on the other devices connected to the session.
+
+    \item \textbf{Each user can temporarily or permanently mute their own audio.} \hfill\\
+        A user must be able to mute the playback when it is connected to a session, which relates to the use cases found in \cref{sec:category_increase_volume_and_area}.

```

Comment:
bare lige for at gøre det klart kunne man skrive at denne er relateret til den forrige.